checkov/terraform/checks/resource/gcp/DataprocPublicIpCluster.py

```diff
@@ -0,0 +1,20 @@
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.enums import CheckCategories
+
+
+class DataprocPublicIpCluster(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure Dataproc Clusters do not have public IPs"
+        id = "CKV_GCP_103"
+        supported_resources = ['google_dataproc_cluster']
+        categories = [CheckCategories.GENERAL_SECURITY]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def get_inspected_key(self):
+        return 'cluster_config/[0]/gce_cluster_config/[0]/internal_ip_only'
+
+    # Accounts for if key is present but is set to False
+    def get_expected_value(self):
+        return True
```

Comment:
@jdyke This isn't needed as it is the default of the parent

Reply:
Oh weird I thought I had to use that in other examples and the same applied here. I removed it, thanks for catching that